docs/src/layouts/DocLayout.vue

```diff
@@ -45,13 +45,14 @@ q-layout.doc-layout(view="hHh LpR lff", @scroll="onScroll")
   q-page-container
     router-view
 
-  q-page-scroller(scroll-offset="150" :offset="[18, 18]")
+  q-page-scroller(:scroll-offset="150" :offset="[18, 18]")
```

Comment:
Why wasn't the normal "scroll-offset" ok here?

Reply:
The scroll-offset prop is expecting a Number, not a String. It was the reason for the many console error logs.